src/TMap.cpp

```diff
@@ -2082,7 +2082,10 @@ int TMap::createMapLabel(int area, QString text, float x, float y, float z, QCol
     QPen lpen;
     lpen.setColor(label.fgColor);
     QFont font;
-    font.setPointSize(fontSize); //good: font size = 50, zoom = 30.0
+    font.setPointSize(fontSize);
+    if (fontName.has_value()) {
```

Comment:
Does it matter if this is specified *after* the size - it might be better to set the family before the size - or actually it could be done like this I *think*:
```cpp
    QFont font(fontName.has_value() ? fontName.value() : QString(), fontSize);
```
on the basis that the Qt Documentation says: "If the family isn't available a family will be set using the font matching algorithm."

Reply:
Improved